.github/workflows/tests.yml

```diff
@@ -60,4 +58,7 @@ jobs:
 
       - name: Run tests
         run: |
-          vendor/bin/phpunit -v
+          vendor/bin/phpunit
```

Comment:
Why you've removed the `-v` option?

Reply:
not needed, is it? already quite verbose

Reply:
The `quite verbose` term can't be measured. If you'd like this option to be removed please show the comparison of both PHPUnit runs (with `-v` and without it) to see what we've loosing by removing this option.